open-banking-accelerator/accelerators/ob-bi/README.md

```diff
@@ -0,0 +1,16 @@
+### **WSO2 Open Banking BI Accelerator 3.0.0**
+
+**Try Locally:**
+
+Prerequisites:
+1. Download WSO2 SI product 
+2. Download the WSO2 Open Banking Business Intelligence Accelerator 
+3. Setup MySQL database server
+4. Install Java on your local machine
+
+Steps:
+1. Extract WSO2 SI <WSO2_BI_HOME>
+2. Extract WSO2 OB BI Accelerator to WSO2_BI_HOME 
+3. Run <WSO2_OB_BI_ACC_HOME>/bin/merge.sh. This will copy the artifacts to the WSO2 SI
+4. Run <WSO2_OB_BI_ACC_HOME>/bin/configure.sh. This will configure the server and create databases.
+5. Run <WSO2_BI_HOME>/bin/server.sh to start the server
```

Comment:
We have used OB_BI and BI both terms. Good if we can stick to one term.

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/e19df9383f6ae2b74becba71cc2e1341a9a489bc